src/tests/scenarios/test_iso_yielder_setting.cairo

```diff
@@ -0,0 +1,731 @@
+    let project = deploy_project(signers.owner);
+    let erc20 = deploy_erc20(signers.owner);
+    let yielder = deploy_yielder(project, erc20, signers.owner);
+    let minter = deploy_minter(project, erc20, signers.owner);
+
+    // Setup
+    setup_project(project, @signers);
+    setup_erc20(erc20, yielder, @signers);
+    setup_yielder(project, erc20, yielder, @signers, price);
+    setup_minter(project, minter, @signers);
+
+    // Setup Minter 
+    set_contract_address(signers.owner);
+    let project_minter = IMinterDispatcher { contract_address: project };
+    project_minter.add_minter(SLOT, minter);
+
+    // Return
+    let contracts = Contracts { project: project, erc20: erc20, yielder: yielder, minter: minter };
+    (signers, contracts)
+}
+
+fn setup_for_apr(n: u64) -> (Signers, Contracts) {
+    // Deploy
+    let signers = Signers { owner: deploy_account('OWNER'), anyone: deploy_account('ANYONE'), anyone2: deploy_account('ANYONE2'), };
+    let project = deploy_project(signers.owner);
+    let erc20 = deploy_erc20(signers.owner);
+    let yielder = deploy_yielder(project, erc20, signers.owner);
+    let minter = deploy_minter(project, erc20, signers.owner);
+
+    // Setup
+    setup_project_apr(project, @signers, n);
+    setup_erc20(erc20, yielder, @signers);
+    // setup_yielder(project, erc20, yielder, @signers);
+    setup_minter(project, minter, @signers);
+
+    // Setup Minter 
+    set_contract_address(signers.owner);
+    let project_minter = IMinterDispatcher { contract_address: project };
+    project_minter.add_minter(SLOT, minter);
+
+    // Return
+    let contracts = Contracts { project: project, erc20: erc20, yielder: yielder, minter: minter };
+    (signers, contracts)
+}
+
+#[test]
+#[available_gas(400_000_000)]
+fn test_yielder_iso_banegas_farm_deposit_withdraw_value() {
+    let (signers, contracts) = setup(PRICE);
+    // Instantiate contracts
+    let farmer = IFarmDispatcher { contract_address: contracts.yielder };
+    let yielder = IYieldDispatcher { contract_address: contracts.yielder };
+    let minter = IMinterDispatcher { contract_address: contracts.project };
+    let project = IProjectDispatcher { contract_address: contracts.project };
+    let absorber = IAbsorberDispatcher { contract_address: contracts.project };
+    let erc3525 = IERC3525Dispatcher { contract_address: contracts.project };
+    let erc20 = IERC20Dispatcher { contract_address: contracts.erc20 };
+
+    // Prank caller as owner
+    set_contract_address(signers.owner);
+
+    // Grant minter rights to owner, mint 1 token to anyone and revoke rights
+    minter.add_minter(SLOT, signers.owner);
+    let token_id = project.mint(signers.anyone, SLOT, VALUE);
+    minter.revoke_minter(SLOT, signers.owner);
+
+    // Setup project value
+    let project_value = erc3525.total_value(SLOT);
+    absorber.set_project_value(SLOT, project_value);
```

Comment:
The project value is already set in the setup, this should be removed as it corrupts the project_value.
This will change the farming values.